manifests/f/FadeIn/FadeIn/4.0.13.1028/FadeIn.FadeIn.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FadeIn.FadeIn
+PackageVersion: 4.0.13.1028
+PackageLocale: en-US
+Publisher: GCC Productions Inc.

```

This must match Control Panel
```suggestion
Publisher: Fade In Professional Screenwriting Software
```